ControlRoomApplication/ControlRoomApplication/Controllers/AASharpControllers/CoordinateCalculationController.cs

```diff
@@ -12,21 +13,43 @@ public CoordinateCalculationController()
 
         }
 
-        public Coordinate CalculateCoordinates(string celestialBody, DateTime date)
+        public Coordinate CalculateCoordinates(Appointment appt)
```

Comment:
This should be altered to return a Dictionary<DateTime, Orientation> as opposed to a single coordinate as that is the main function of the CoordinateCalculationController, this was established last semester but not implemented. See my slack message for more detail